Blockzilla/SearchSuggestionsPromptView.swift

```diff
@@ -0,0 +1,124 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import SnapKit
+
+protocol SearchSuggestionsPromptViewDelegate: class {
+    func searchSuggestionsPromptView(_ searchSuggestionsPromptView: SearchSuggestionsPromptView, didEnable: Bool)
+}
+
+class SearchSuggestionsPromptView: UIView {
+    weak var delegate: SearchSuggestionsPromptViewDelegate?
+    static let respondedToSearchSuggestionsPrompt = "SearchSuggestionPrompt"
+    private let buttonBorderMiddle = UIView()
+    private let buttonBorderTop = UIView()
+    private let disableButton = InsetButton()
+    private let enableButton = InsetButton()
+    private let promptContainer = UIView()
+    private let promptCornerRadius: CGFloat = 12
```

Comment:
nit: magic number, can we declare this in UIConstants (or use a value that already exists in it?)